CHANGELOG.md

```diff
@@ -6,6 +6,16 @@ All notable changes to [camunda-bpmn-js](https://github.com/camunda/camunda-bpmn
 
 ___Note:__ Yet to be released changes appear here._
 
+## 4.12.2
```

Comment:
This is going to be a minor (feature) release since we are adding a feature: 
https://github.com/camunda/camunda-bpmn-js/pull/372/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR16

Reply:
```suggestion
## 4.13.0
```